5watchdog

```diff
@@ -61,6 +61,7 @@ echo "Watchdog for $nvOC_Ver"
 echo "Version: $nvOC_wdog_ver"
 echo ""
 
+export NVOC="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I would remove this line for the same reasoning discussed elsewhere, but take this as good general guideline and nothing more.